TRANSLATION.md

```diff
@@ -87,3 +87,9 @@ Translation Keys should not be translated during Unit Testing. If your Component
 - Unit Tests should not rely on text, titles, or string bags, as these texts will change arbitrarily and make the test suite fail.
   - In this case, you should test your component by aria-text, or other `aria-*` attributes or even by class names or other artifacts.
 - Visual Regression Testing is recommended to test how different languages and text appear within a Component.
+
+## Deploying Translations
```

Comment:
maybe add link/ref to the right workflow ?

Reply:
i had thought about that, but fresh in my mind was renaming the files, and i didn't want to invite broken links. people motivated to find it will browse IMO

Reply:
Okay ! Let's not link it. But why not explain that a GH action now it's only say that a cron schedule